src/main/java/de/tum/in/www1/artemis/service/learningpath/LearningPathNgxService.java

```diff
@@ -262,7 +263,7 @@ private void generateNgxPathRepresentationForCompetency(User user, CourseCompete
         currentCluster.add(NgxLearningPathDTO.Node.of(startNodeId, NgxLearningPathDTO.NodeType.COMPETENCY_START, competency.getId()));
         currentCluster.add(NgxLearningPathDTO.Node.of(endNodeId, NgxLearningPathDTO.NodeType.COMPETENCY_END, competency.getId()));
 
-        final var recommendedLearningObjects = learningPathRecommendationService.getRecommendedOrderOfLearningObjects(user, competency, state);
+        final var recommendedLearningObjects = learningPathRecommendationService.getRecommendedOrderOfLearningObjects(user, competency, state, new LearningPathsConfiguration());
```

Comment:
Is this right? Wouldn't we use the course's learning path configuration here?

Reply:
This is legacy code that is no longer used and will be removed soon, so it's fine